publish/publish.proj

```diff
@@ -65,6 +65,14 @@
                    Condition="'@(_MissingBlobNames)' == ''" />
     <Message Importance="High" Condition="'@(_MissingBlobNames)' != ''" Text="Skipping finalization, missing these blobs: '@(_MissingBlobNames)'" />
   </Target>
+  
+  <Target Name="UpdateVersionsRepo"
```

Comment:
That's reasonable.  I'll ask @karajas to send me the manual command / tokens for this.